src/atdf/patch.rs

```diff
@@ -40,7 +40,7 @@ pub fn signals_to_port_fields(chip: &mut chip::Chip, tree: &xmltree::Element) ->
             .collect();
 
         for reg in port.registers.values_mut() {
-            if !reg.name.ends_with(name) {
+            if !correct_reg_name(&reg.name, name) {
                 log::error!("Register {:?} has a weird name!", reg.name);
             }
```

Comment:
For registers which do not have an allowed name, we don't want to error out.  The other registers should be skipped silently.